app/sidekiq/central_mail/submit_form4142_job.rb

```diff
@@ -131,20 +136,39 @@ def lighthouse_service
       @lighthouse_service ||= BenefitsIntakeService::Service.new(with_upload_location: true)
     end
 
-    def upload_to_lighthouse
-      Rails.logger.info(
-        'Successful Form4142 Submission to Lighthouse',
-        { benefits_intake_uuid: lighthouse_service.uuid, submission_id: @submission_id }
-      )
-
-      payload = {
-        upload_url: lighthouse_service.location,
+    def payload_hash(lighthouse_service_location)
+      {
+        upload_url: lighthouse_service_location,
         file: { file: @pdf_path, file_name: @pdf_path.split('/').last },
         metadata: generate_metadata.to_json,
         attachments: []
       }
+    end
 
+    def upload_to_lighthouse
+      Rails.logger.info(
+        'Successful Form4142 Upload Intake UUID aquired from Lighthouse',
+        { benefits_intake_uuid: lighthouse_service.uuid, submission_id: @submission_id }
+      )
+
+      payload = payload_hash(lighthouse_service.location)
       lighthouse_service.upload_doc(**payload)
+
+      log_info = { benefits_intake_uuid: lighthouse_service.uuid, submission_id: @submission_id }
+
+      if Flipper.enabled?(POLLING_FLIPPER_KEY)
+        form526_submission = Form526Submission.find(@submission_id)
+        form_submission    = FormSubmission.create(
+          form_type: FORM4142_FORMSUBMISSION_TYPE, # form526_form4142
+          benefits_intake_uuid: lighthouse_service.uuid,
+          form_data: '{}', # we have this already in the Form526Submission.form['form526']['form526']['form4142']
```

Comment:
I think `Form526Submission.form['form4142']` will return form4142 data.

Reply:
`.form['form526']['form526']['form4142']` will, but we already have this data in the related records, I am not convinced we need to duplicate this data into here. This is here because other teams stuff ONLY keep this data in this place. If I can avoid putting duplicate PII data in (even though it is encrypted, so it is fine) I think I want to. 

I am not super firm on this. You could probably convince me it may be better to duplicate this data into here instead of making two DB calls to get it if we need it later with this record.

Reply:
Valid. I just thought 4142 data was accessed via `.form['form4142']`, not buried within `['form526']['form526']`.